elasticdl/python/ps/parameters.py

```diff
@@ -99,6 +103,8 @@ def init_from_model_pb(self, model_pb):
             self._init_embedding_params(embeddings_pb)
             self.version = model_pb.version
             self.init_status = True
+            return True
```

Comment:
Combine the two return statements to one return `self.init_status` instead?

Reply:
> Combine the two return statements to one return `self.init_status` instead?

The return value does not equal to `init_status`, it means whether `init_from_model_pb` accepts this model pb. I will add a docs here.